crypto/objects/obj_mac.num

```diff
@@ -1117,3 +1117,4 @@ RSA_SHA3_224		1116
 RSA_SHA3_256		1117
 RSA_SHA3_284		1118
 RSA_SHA3_512		1119
+scrypt		1120
```

Comment:
Well spotted. I've reverted

```
    crypto/asn1/charmap.h \
    crypto/conf/conf_def.h \
    crypto/objects/obj_dat.h \
    crypto/objects/obj_mac.num \
    crypto/objects/obj_xref.h \
    crypto/objects/obj_xref.txt \
    include/openssl/obj_mac.h
```

and ran ``make update``. Can confirm that this is a leftover from the previous runs that became obsolete. b7c9b69 fixes this.